serialmfg/serial_resources/part_number.py

```diff
@@ -0,0 +1,91 @@
+"""
+This file contains the PartNumber class and the PartNumbers class.
+"""
+from ..api_client import APIClient
+from ..exceptions import SerialAPIException
+class PartNumberNotFound(SerialAPIException):
+    """
+    Exception for when a Part Number is not found
+    """
+    pass
+
+class PartNumbers:
+    """
+    A class for part number data methods
+    """
+    @staticmethod
+    def get(part_number, component_id=None):
+        """
+        Get part number data
+
+        Args:
+        - part_number (str): Part Number
+        - component_id? (str): Component ID - may be needed if part number is not unique
+
+        Returns:
+        - A PartNumber object
+        """
+        client = APIClient()
+        query_params = {
+            'part_number': part_number
+        }
+        response = client.make_api_request('/part-numbers', 'GET', query_params)
+        if not response:
+            raise PartNumberNotFound('Part Number not found')
+        return PartNumber(response[0])
```

Comment:
We say "Component ID - may be needed if part number is not unique" but we default to returning only a single part number behind the scenes. In other words, a user would never know it's not unique. What do you think about throwing an error and forcing the user to be specific if there's ambiguity?